cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -89,21 +95,29 @@
     protected boolean isMetricsEnabled;
 
     protected JsonObject metricsConfig;
-    protected String metricsConfigName;
+    protected String metricsAndLogsConfigName;
```

Comment:
I still prefer using a more generic term like 'ancillary' to the more specific 'metricsAndLogs', because in the future it's possible that we use the same mechanism here for adding other things to the pod's fs (e.g. JAAS configs etc).